public/content/documentation/web/page-level/navigation-landmark.md

```diff
@@ -1,68 +1,48 @@
-
-   &mdash; **Doubletap:** This typically activates most elements.
-
+   - Swipe: Focus moves within the nav
+   - Doubletap: This typically activates most elements
 3. Listen to screenreader output on all devices
-
-   &mdash; **Role:** It is discoverable with screenreader shortcuts as a navigation landmark.
-
-   &mdash; **Name:** It indicates its role and, if multiple navigations are present (e.g., Main navigation, Site map, Breadcrumbs), their respective names.
+   - **Name**: It indicates its role AND IF multiple navigations are present (ex: Main navigation, Site map, Breadcrumbs), the name of the navigation
+   - **Role**: It is discoverable with screenreader shortcuts as a navigation landmark
 
 Full information: https://www.magentaa11y.com/#/web-criteria/page-level/navigation-landmark
 
 ## Gherkin
 
-### a11y - Web Accessibility Acceptance Criteria
+### #a11y - Web Accessibility Acceptance Criteria
 
 How to test a navigation landmark
 
 GIVEN THAT I am on a page with a navigation landmark
 
-1. **Keyboard for mobile & desktop**
-
-   &mdash; WHEN I use the arrow keys to browse the navigation menu
-
-   &mdash; I SEE the screen scrolls through the page.
-
-2. **Desktop screenreader**
-
-   &mdash; WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND
-
-   &mdash; I use the arrow keys to browse the navigation menu
-
-   &mdash; I HEAR It indicates its role and, if multiple navigations are present (e.g., Main navigation, Site map, Breadcrumbs), their names.
-
-   &mdash; I HEAR It is discoverable with screenreader shortcuts as a navigation landmark.
-
-3. **Mobile screenreader**
-
-   &mdash; WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
-
-   &mdash; I swipe to focusable elements in the navigation
-
-   &mdash; I HEAR It indicates its role and, if multiple navigations are present, their names.
-
-   &mdash; I HEAR It is discoverable with screenreader shortcuts as a navigation landmark.
+1. Keyboard for mobile & desktop
+   - WHEN I use the arrow keys to browse the navigation menu, I SEE the screen scrolls through the page.
+  
+2. Desktop screenreader
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND
+   - I use the arrow keys to browse the navigation menu
+     - I HEAR It indicates its role AND IF multiple navigations are present (ex: Main navigation, Site map, Breadcrumbs), the name of the navigation
+     - I HEAR It is discoverable with screenreader shortcuts as a navigation landmark.
+
+3. Mobile screenreader
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
+   - I swipe to focusable elements in the navigation
+     - I HEAR It indicates its role AND IF multiple navigations are present (ex: Main navigation, Site map, Breadcrumbs), the name of the navigation
+     - I HEAR It is discoverable with screenreader shortcuts as a navigation landmark.
 
 Full information: https://www.magentaa11y.com/#/web-criteria/page-level/navigation-landmark
 
```

Comment:
Under Code examples > Use semantic HTML, the code snippet is missing the first two anchor links in the snippet. 

Additionally, the mockup example of the code snippet is not included in the new MagentaA11Y.

Reply:
Thank you for noticing I missed part of the code snippet!

What do you think about the value of the live HTML in this case? I wasn't sure.